src/site/content/en/angular/getting-started-with-server-side-rendering-in-angular/index.md

```diff
@@ -0,0 +1,150 @@
+---
```

Comment:
The URLs for these posts are pretty long. Might want to consider shortening them.